books/free-programming-books-langs.md

```diff
@@ -971,12 +972,14 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Practical Series: A website template](https://practicalseries.com/1001-webdevelopment/) - Michael Gledhill (HTML)
 * [Pro HTML5 Programming](https://web.archive.org/web/20181215200026/http://apress.jensimmons.com/v5/pro-html5-programming/ch0.html) - Jen Simmons, Chris O'Connor, Dylan Wooters, Peter Lubbers *(:card_file_box: archived)*
 * [Resilient Web Design](https://resilientwebdesign.com/#Resilientweb%20design) - Jeremy Keith
+* [Rote Learning HTML & CSS](https://meiert.com/de/publications/books/rote-learning-html-and-css/) – Jens Oliver Meiert
```

Comment:
I think this should be on the German books page. Also, take a look at how we list other Leanpub books with 0 cost - still needs a login, and thus an access note.

Reply:
Good catch—corrected to go to the English landing page. The book is available on more platforms, including GitHub, and for free everywhere.

Reply:
(Hope that also addresses the Leanpub point.)